deepchem/molnet/load_function/pdbbind_datasets.py

```diff
@@ -1,458 +1,164 @@
-        feature_types=[
-            'ecfp', 'splif', 'hbond', 'salt_bridge', 'pi_stack', 'cation_pi',
-            'charge'
-        ],
-        flatten=True)
-  elif featurizer == "atomic":
-    # Pulled from PDB files. For larger datasets with more PDBs, would use
-    # max num atoms instead of exact.
-    frag1_num_atoms = 70  # for ligand atoms
-    frag2_num_atoms = 24000  # for protein atoms
-    complex_num_atoms = 24070  # in total
-    max_num_neighbors = 4
-    # Cutoff in angstroms
-    neighbor_cutoff = 4
-    featurizer = ComplexNeighborListFragmentAtomicCoordinates(
-        frag1_num_atoms, frag2_num_atoms, complex_num_atoms, max_num_neighbors,
-        neighbor_cutoff)
+    # Extract labels
+    with open(index_labels_file, "r") as g:
+      labels = np.array([
+          # Lines have format
+          # PDB code, resolution, release year, -logKd/Ki, Kd/Ki, reference, ligand name
+          # The base-10 logarithm, -log kd/pk
+          float(line.split()[3]) for line in g.readlines() if line[0] != "#"
+      ])
+
+    return (protein_files, ligand_files, labels, pdbs)
+
+
+def load_pdbbind(
+    featurizer: dc.feat.ComplexFeaturizer,
+    splitter: Union[dc.splits.Splitter, str, None] = 'random',
+    transformers: List[Union[TransformerGenerator, str]] = ['normalization'],
+    reload: bool = True,
+    data_dir: Optional[str] = None,
+    save_dir: Optional[str] = None,
+    set_name: str = 'refined',
+    **kwargs
+) -> Tuple[List[str], Tuple[Dataset, ...], List[dc.trans.Transformer]]:
+  """Load PDBBind dataset.
+
+  The PDBBind dataset includes experimental binding affinity data
+  and structures for 4852 protein-ligand complexes from the "refined set"
+  and 12800 complexes from the "general set" in PDBBind v2019.
+  The refined set removes data with obvious problems
+  in 3D structure, binding data, or other aspects and should therefore
+  be a better starting point for docking/scoring studies. Details on
+  the criteria used to construct the refined set can be found in [4]_.
+  The general set does not include the refined set.
+
+  Random splitting is recommended for this dataset.
+
+  The raw dataset contains the columns below:
+
+  - "ligand" - SDF of the molecular structure
+  - "protein" - PDB of the protein structure
+  - "CT_TOX" - Clinical trial results
 
-  else:
-    raise ValueError("Featurizer not supported")
-  print("Featurizing Complexes")
-  features, failures = featurizer.featurize(ligand_files, protein_files)
-  # Delete labels for failing elements
-  labels = np.delete(labels, failures)
-  dataset = deepchem.data.DiskDataset.from_numpy(features, labels)
-  # No transformations of data
-  transformers = []
-  if split == None:
-    return pdbbind_tasks, (dataset, None, None), transformers
+  Parameters
+  ----------
+  featurizer: Featurizer or str
+    the featurizer to use for processing the data.  Alternatively you can pass
+    one of the names from dc.molnet.featurizers as a shortcut.
```

Comment:
This description needs to be updated for this class.  It has to be a ComplexFeaturizer.

Reply:
Thanks for this catch, @peastman! I'll fix this and then merge it in.